app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,5 @@
     <string name="user_name_text">Username</string>
     <string name="password_text">Password</string>
     <string name="login_text">Login</string>
+    <string name="log_me_in">Log Me IN</string>
```

Comment:
Please create separate pull-request for string changes.

Reply:
thank you